heron/common/src/cpp/network/network_error.h

```diff
@@ -30,4 +33,12 @@ enum NetworkErrorCode {
   TIMEOUT
 };
 
+static inline std::ostream& operator<<(std::ostream& os, const NetworkErrorCode code) {
+  std::string strs[] = { "OK", "Connection error", "Read error", "Write error",
```

Comment:
just pointing out. i have no opinion since C++ language does not have the capability of allowing people to use enum reflection easily.